specs/error_state/ErrorCodeStore.md

```diff
@@ -1,41 +1,30 @@
 # ErrorCodeStore state
 
 ## Procedure
-`ErrorCodeStore` is for two original code store related errors: `CodeStoreOutOfGas` and 
-`MaxCodeSizeExceeded`.
-this type of error only occurs when executing create(create,create2) op code or tx deploy 
-transaction(tx.to = null).
+`ErrorCodeStore` is for two original code store related errors: `CodeStoreOutOfGas` and `MaxCodeSizeExceeded`. This type of error only occurs when executing create(create,create2) op code or tx deploy transaction(tx.to = null).
```

Comment:
I know this was not changed in this PR, but perhaps a bit of language cleaning could be done in this PR? Like "op code" -> "opcode" or perhaps deleting "tx" (I think it's redundant).

Reply:
fixed in [6ad3b04](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/475/commits/6ad3b041f054e6bed0b1a292bcaacc6e95cd6ecf)

Reply:
I was hesitanting to refine the sentence or not, but I agree with you, we should do language cleaning in this PR. Have a fix, [6ad3b04](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/475/commits/6ad3b041f054e6bed0b1a292bcaacc6e95cd6ecf), please help review it's more clear for you or not.